src/main/java/io/confluent/connect/elasticsearch/ElasticsearchSinkConnectorConfig.java

```diff
@@ -132,7 +133,10 @@ protected static ConfigDef baseConfigDef() {
                   group, ++order, Width.LONG, "Topics for 'Ignore Key' mode")
           .define(TOPIC_SCHEMA_IGNORE_CONFIG, Type.LIST, "", Importance.LOW,
                   "List of topics for which ``" + SCHEMA_IGNORE_CONFIG + "`` should be ``true``.",
-                  group, ++order, Width.LONG, "Topics for 'Ignore Schema' mode");
+                  group, ++order, Width.LONG, "Topics for 'Ignore Schema' mode")
+          .define(BEHAVIOR_ON_NULL_VALUES_CONFIG, Type.STRING, "ignore", ConfigDef.ValidString.in("ignore", "delete", "fail"), Importance.LOW,
+                  "How to handle records with non-null keys but null values. Valid options are 'ignore', 'delete', and 'fail'.",
```

Comment:
Nit on the grammar of "and" rather than "but": I'd suggest "How to handle records with a non-null key and null value, such as a tombstone record. Valid options ...".

Reply:
Any thoughts on

>How to handle records with a non-null key and a null value (i.e. Kafka tombstone records). Valid options are 'ignore', 'delete', and 'fail' (case-insensitive).

Reply:
Sounds good to me. I wouldn't add the "(case-insensitive)" part. 

We don't care to encourage diversion here. We just want to support broadly this property so we don't have issues or questions on the mailing list.